bin/verify_release_assets

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
+
+set -e
+
+. libstd-ecm.sh
+
+: "${REPOSITORY:=rancher/rke2}"
+: "${STRICT_MATCH:=false}"
+: "${LIST_ASSETS:=false}"
+
+usage() {
+    echo "usage: $0 [rslh] <repository>
+    -l    list assets file names
+    -r    target repository. default: $REPOSITORY
+    -s    strict release tag name match
+    -h    show help
+
+examples:
+    $0 -s v1.23.4
+    $0 -s v1.23.4-rc2+rke2r1
+    $0 -l -r rancher/rke2 v1.23.4"
+}
+
+print_status() {
+    release=${1}
+    current_count=${2}
+    expected=''
+
+    case $REPOSITORY in
+    *rke2)
+        expected="30"
+        ;;
+    *rke2-packaging)
+        expected="23"
+        ;;
+    *k3s)
+        expected="18"
+        ;;
+    *)
+        printf '-'
+        return
+        ;;
+    esac
+
+    printf "release: %s expected: %s got: %s status: " "$release" "$expected" "$current_count"
+
+    if [ "$current_count" = "$expected" ]; then
+        print_green 'OK'
+    else
+        print_red 'MISMATCH'
+    fi
+
+    echo
```

Comment:
Is this to get a new line after the status output? If so, you can add "\n" to the input of the color function.

Reply:
I mean something like this:

```sh
print_green "OK\n"
```

Reply:
Oh, I tried that as well and prints `\n` literally, let me double check `printf` docs